crates/biome_js_analyze/src/aria_analyzers/a11y/use_valid_aria_role.rs

```diff
@@ -75,74 +73,14 @@ declare_rule! {
     }
 }
 
-#[derive(Default, Deserialize, Serialize, Eq, PartialEq, Debug, Clone)]
+#[derive(Clone, Debug, Default, Deserialize, Deserializable, Eq, PartialEq, Serialize)]
 #[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 pub struct ValidAriaRoleOptions {
-    allowed_invalid_roles: Vec<String>,
+    allow_invalid_roles: Vec<String>,
```

Comment:
Why the name of the property was changed? This will break user config?

Reply:
The JSON property it maps to is `allowInvalidRoles`, so I changed the name in the struct to match it. This should actually prevent breaking the user config.